tests/test_carbonlink.py

```diff
@@ -116,6 +116,14 @@ def test_chr_get_nodes(self):
         ]
         self.assertEqual(node, expected)
 
+    def test_chr_break_get_node(self):
+        hosts = [
+            ("127.0.0.1",None)
+        ]
+        carbonlink = ConsistentHashRing(hosts)
+        node = carbonlink.get_node('hosts.worker44.cpu')
```

Comment:
Hm, it looks like this passed the automated test... I arrived upon this value by iterating through metric values (e.g. `for x in range(100): ... 'hosts.worker%s.cpu' % x ... `).